.github/pull_request_template.md

```diff
@@ -0,0 +1,43 @@
+Thank you for contributing to this repository!
+
+You can help us review your changes by answering these questions.
+They are all optional, but the more information you provide the easier it will be for us to review your changes.
+Everyone here is a volunteer, so please help us out if you can.
+
+If you want to discuss anything Jupyter related, or to meet other users and developers, please say hello on https://discourse.jupyter.org/ .
+
+
+### What does this PR do?
+<!--
+Please summarise this PR.
+Please also edit the PR title so that it contains enough context to go into a changelog.
+Use "Fixes #<NUM>" if this fixes an existing issue.
+It may help to list each change with an explanation of why it's needed- remember that what seems obvious to you may not be obvious to a reviewer.
+-->
+
+### Is this PR related to another issue, or is it part of a larger body of work?
```

Comment:
```suggestion
### Is this PR related to an issue, or is it part of a larger body of work?
```

This part here makes me think that this should be the place where one would link the issue associated with this PR.
If so, I was thinking that it might be useful to add a tip in this section description about using keywords like `closes / fixes #issueNo` to automatically close the issue.
What do you think?